docs/Administration.rst

```diff
@@ -2366,14 +2393,44 @@ Managing memberships
 Select the Memberships tab. Here, you can manage user roles in a
 specific group.
 
-By default, 9 membership types are available in PRODUCT including
-Member, Author, Editor, Manager, Redactor, Validator, Webdesigner,
-Publisher and \*.
+By default, 9 membership types are available in eXo Platform including
+Member, Author, Editor, Manager, Redactor, Validator, Webdesigner, Publisher and \*.
 
 |image178|
 
+The membership, such as `manager:/platform/users`, is defined by both membership type (manager, member, author...) 
+and a group i.e. it is the combinaison between a group and a membership type to define the user's role in the group.
+
 .. note:: The \* membership is identical to any other type, so choosing this type means adding all available membership types.
 
+**Membership usage in eXo Platform**
+
+In eXo Platform, the most used memships are `manager`, `member` and `*`.
+
+- The `manager` role to define users with managing privileges
+- The `member` role to define normal users in a group
+- `*` all the roles combined, that means a user with * membership type will have the permissions assigned to all  membership types within this group
+
+Other roles i.e. Author, Editor, Redactor, Validator, Webdesigner and Publisher are mainly used for the 
+:ref:`publication process <PublicationProcess>` which could be 
+:ref:`customized <PLFDevGuide.DevelopingContent.PublicationLifecycle.CreatingPublicationLifecycle>`.
+
+The most used roles are:
+
+- The `publisher` role defines users allowed to :ref:`publish content <PublishContent>` i.e. they are able to switch
+  contents from staged to published status.
+  
+- The `Author` role defines users able to add and manage content in draft, pending or approved statuses.
+
+Remaining roles could be used when you create your 
+:ref:`own publication lifecycle <PLFDevGuide.DevelopingContent.PublicationLifecycle.CreatingPublicationLifecycle>`.
+
+.. note:: For :ref:`Spaces groups <spacesGroups>`, a user belonging to a space could be either a manager or a member.
```

Comment:
I think news is adding more roles in spaces around news publication. @samehaidi ?